src/get-started/java-binding.md

```diff
@@ -246,16 +272,16 @@ processing on the node.
 
 The external service API is used for the interaction between the service and the
 external systems.
-A set of operations is defined by each service and can include sending
-transactions to the network, read requests for blockchain data with the
-provision of corresponding cryptographic proof, etc. Exonum provides an embedded
-web framework for implementing the REST-interface of the service.
+A set of operations defined by a service usually includes read requests
+for blockchain data with the provision of corresponding cryptographic proof. 
```

Comment:
for the blockchain data with the provision of the corresponding cryptographic proof.